spec/datadog/core/configuration/components_spec.rb

```diff
@@ -1050,7 +1075,7 @@
         end
 
         context 'on Ruby 3.x' do
-          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION < '3' }
+          before { skip 'Behavior does not apply to current Ruby version' if RUBY_VERSION < '3.' }
```

Comment:
Why not `3.0`?

Reply:
Either is fine! Changed in 482d94643fc45c023482d38938380d85d3107e99